src/os_handlers.eliom

```diff
@@ -25,6 +25,7 @@
   open Eliom_content.Html.F
 ]
 
+(* Set personal data *)
```

Comment:
When we have a good name that accurately describes the functionality, there is no need to duplicate the description as a comment :). But no need to remove these now.